packages/vue/package.json

```diff
@@ -8,17 +8,17 @@
   ],
   "version": "0.5.1",
   "scripts": {
-    "serve": "yarpm create-index-files && vue-cli-service serve",
-    "build": "yarpm create-index-files && vue-cli-service build --target lib --name storefront-ui ./src/index.js",
+    "serve": "yarn create-index-files && vue-cli-service serve",
+    "build": "yarn create-index-files && vue-cli-service build --target lib --name storefront-ui ./src/index.js",
     "lint": "vue-cli-service lint",
     "lint-staged": "lint-staged",
     "test:unit": "vue-cli-service test:unit",
     "test:dev": "vue-cli-service test:unit --watch",
-    "storybook:serve": "yarpm create-index-files && vue-cli-service storybook:serve -s ./public -p 6006 -c config/storybook",
-    "storybook:build": "yarpm create-index-files && vue-cli-service storybook:build -s ./public -c config/storybook",
+    "storybook:serve": "yarn create-index-files && vue-cli-service storybook:serve -s ./public -p 6006 -c config/storybook",
+    "storybook:build": "yarn create-index-files && vue-cli-service storybook:build -s ./public -c config/storybook",
     "docs:vuecomponents": "node scripts/create-vue-components-docs.js",
-    "docs:dev": "yarpm docs:vuecomponents && vuepress dev docs",
-    "docs:build": "yarpm docs:vuecomponents && vuepress build docs",
+    "docs:dev": "yarn docs:vuecomponents && vuepress dev docs",
+    "docs:build": "yarn docs:vuecomponents && vuepress build docs",
     "create-index-files": "node scripts/create-index-files.js",
     "postinstall": "node scripts/postinstall.js && yarn create-index-files",
```

Comment:
I think the `postinstall` command also runs when someone _uses_ SFUI in their projects. If that someone uses npm instead of yarn that line would fail. But we can circumvent this problem by directly calling `&& node scripts/create-index-files.js` here instead of calling the indirected command target.